config/webpack/webpack.config.js

```diff
@@ -114,7 +115,7 @@ const webpackConfig = (options, env, argv) => {
         log: false,
         plugins: [
           PostCSSCustomProperties({
-            preserve: true,
+            preserve: !disableCSSCustomProperties,
```

Comment:
nitpick - we only use this here, want to directly use the variable, instead of defining it all the way at the top?
```suggestion
            preserve: !env.disableCSSCustomProperties,
```

Reply:
I'm going to leave this as is or now.